### DISCUSSION TITLE
video grey/black no audio

### DISCUSSION THREAD

**Discussion by kareig** [aclap-dev/video-downloadhelper#1076] [Help and Q&A]
### Your question

it was working fine until recently i wanted to download some new videos and they now downloaded but not playable as they are grey/black with no audio.

i have debug log and chrome extension settings attached.
[chrome extension settings.txt](https://github.com/user-attachments/files/16457490/chrome.extension.settings.txt)
[vdh-logs.txt](https://github.com/user-attachments/files/16457491/vdh-logs.txt)

### Addon & CoApp version and details

attached.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

That means I'm pretty sure I can't interpret your log properly.  But I believe I managed to find this URL in your log:

https://epika.lrt.lt/animaciniai-serialai,474/bluji-odcinki,665921/odcinek-31,S01E31,728805

It looks to me like a web site with cartoons but there's a button there saying that I need to join to watch.  Oh well.

On balance, from your description of the symptoms, I'm going to guess that you are hitting DRM protection.  VDH sometimes downloads content that is protected by DRM.  That is a wasted exercise because protected content is scrambled & cannot be played back after downloading.  That's the intended purpose of DRM protection.  The content author is protecting his intellectual property from copying.  You can play it in a player on a web page but you are not permitted to download it.  That is what DRM protection is meant to do.  It sounds like that's what you're hitting.  Y0u might search this forum for other discussions of DRM to see whether you can get any other advice on the subject.

That's my best guess . . . as just another dumb user.
